db/migrate/20231204171438_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: {to_table: :users}
+      t.string :title
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+
+      t.timestamps
+    end
+
+    add_index :posts, :author_id
```

Comment:
- [ ] ActiveRecord creates indexes automatically when you create a reference, so please delete this line because it raises an error while migrating. (see the image below for more info)

![image](https://github.com/martinkarugaba/blog_app/assets/68796232/ce038b51-fc01-4ce4-b554-95235a8b2b15)